state/protocol/badger/state.go

```diff
@@ -394,8 +394,22 @@ func OpenState(
 	}
 	state := newState(metrics, db, headers, seals, results, blocks, setups, commits, statuses)
 
+	finalSnapshot := state.Final()
+
+	// update current epoch phase metric
+	err = state.updateCurrentEpochPhase(finalSnapshot)
+	if err != nil {
+		return nil, fmt.Errorf("failed to update current epoch phase: %w", err)
+	}
+
+	// update current epoch counter metric
+	err = state.updateCurrentEpochCounter(finalSnapshot)
+	if err != nil {
+		return nil, fmt.Errorf("failed to update current epoch counter: %w", err)
+	}
+
 	// update committed final view metric
-	err = state.updateCommittedEpochFinalView(state.Final())
+	err = state.updateCommittedEpochFinalView(finalSnapshot)
```

Comment:
I think it would be better to just keep all the epoch metric methods together in one -- they are always called together anyway, have similar context, and aren't especially long.

Eg:
```go
err := state.updateEpochMetrics(state.Final())
```

Reply:
We also need to make sure we set these metrics on bootstrap in addition to on open ([line 106](https://github.com/onflow/flow-go/pull/905/files#diff-bf93c45a15b46de12072db8fb5962e3a8d4d6abc6adde04ea09d88ce6c639b61R106)).